backend/open_webui/routers/retrieval.py

```
@@ -1382,24 +1414,79 @@ def _compose_text(doc):
-        if progress_callback and chunk_counts:
-            for chunk_file_id, count in chunk_counts.items():
-                progress_callback(
-                    UploadProgress(
-                        stage=UploadStage.VECTORS_PERSISTED,
-                        file_id=chunk_file_id,
-                        collection_name=collection_name,
-                        total_chunks=count,
-                        processed_chunks=count,
+                try:
+                    future.result()
+                except Exception as exc:  # noqa: BLE001 - aggregation happens below
```

Ruff (0.13.1)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)